src/docs/replit/index.md

```diff
@@ -3,16 +3,18 @@
 ## Creating a project
 Follow the instructions in the [Replit Workspace Guide](https://docs.replit.com/programming-ide/introduction-to-the-workspace#how-to-create-a-repl) on how to create a project.
 
+<!-- You need replit core, since they have a limit to 3 workspaces on free account, and deployment needs to be purchased also -->
 ## Connect your repl to your is-a.dev subdomain
-Follow the instructions in the [Replit Custom Domains Guide](https://docs.replit.com/hosting/custom-domains#connecting-your-domain-to-your-repl).
-
-Only follow the "Connecting your domain to your repl" section, then return to this guide for the next steps.
+Since Replit Workspaces has changed alot, follow the instructions in the [Replit Deployment Custom Domains Guide](https://docs.replit.com/cloud-services/deployments/custom-domains). You may need the Replit Deployment subscription to do so
 
 ## Creating the domain file
-Create a JSON file inside `domains` directory (`domains/<subdomain>.json`) with the following content:
+After clicking "Link a Domain" button on your Replit Deployment menu and already typed the domain, you will see this
+![image](https://github.com/user-attachments/assets/32abb589-5fad-4383-8554-eebe005052e9)

```

Comment:
Please upload the image locally

Reply:
On the same `replit` folder?

Reply:
No, you upload the image at the `img` directory and then change this file to point to that image

Reply:
Oh sh*t, it turns out on the `img` folder, i will resolve this